fission-web-server/library/Fission/Web/Server/IPFS/Cluster/Error/Types.hs

```diff
@@ -0,0 +1,30 @@
+module Fission.Web.Server.IPFS.Cluster.Error.Types
+  ( ErrorBody (..)
+  , Error (..)
+  ) where
+
+import Fission.Prelude
+
+data ErrorBody = ErrorBody {message :: String}
```

Comment:
(I know that this is all WIP)

With a single field, it may as well be a `newtype`. Also don't forget to give it a `Show` and `Display` instance :)